drivers/i3c/i3c_npcx.c

```diff
@@ -2232,16 +3020,27 @@ static const struct i3c_driver_api npcx_i3c_driver_api = {
 		.common.dev_list.i2c = npcx_i3c_i2c_device_array_##id,                             \
 		.common.dev_list.num_i2c = ARRAY_SIZE(npcx_i3c_i2c_device_array_##id),             \
 		.pincfg = PINCTRL_DT_INST_DEV_CONFIG_GET(id),                                      \
+		.instance_id = DT_INST_PROP_OR(id, instance_id, 0),                                \
 		.clocks.i3c_pp_scl_hz = DT_INST_PROP_OR(id, i3c_scl_hz, 0),                        \
 		.clocks.i3c_od_scl_hz = DT_INST_PROP_OR(id, i3c_od_scl_hz, 0),                     \
 		IF_ENABLED(CONFIG_I3C_NPCX_DMA, (                                                  \
 			.mdma_clk_subsys = NPCX_DT_CLK_CFG_ITEM_BY_IDX(id, 2),                     \
 		))                                                                                 \
-		IF_ENABLED(CONFIG_I3C_NPCX_DMA, (                                                  \
+				     IF_ENABLED(CONFIG_I3C_NPCX_DMA, (                             \
 			.mdma_base = (struct mdma_reg *)DT_INST_REG_ADDR_BY_IDX(id, 1),            \
-		))                                                                                 \
+		)) };                                                                              \
+	static struct npcx_i3c_data npcx_i3c_data_##id = {                                         \
+		.common.ctrl_config.is_secondary = DT_INST_PROP_OR(id, secondary, false),          \
+		.config_target.static_addr = DT_INST_PROP_OR(id, static_address, 0),               \
+		.config_target.pid = ((uint64_t)DT_INST_TGT_PID_PROP_OR(id, tgt_pid, 0) << 32) |   \
+				     DT_INST_TGT_PID_PROP_OR(id, tgt_pid, 1),                      \
+		.config_target.pid_random = DT_INST_TGT_PID_RAND_PROP_OR(id, tgt_pid, 0),          \
+		.config_target.bcr = DT_INST_PROP_OR(id, bcr, 0),                                  \
```

Comment:
you have `bcr` set with required: true in the yaml, but yet this is with `DT_INST_PROP_OR` which implies it could not be set in the dts. Is this your intent?

Reply:
Done.
Update `instance_id` and `bcr `  with `DT_INST_PROP`.